views/app/list.jade

```diff
@@ -1,5 +1,5 @@
 mixin taskText
-  markdown.task-text(text=' ::task.text', remove-watch='true')
+  markdown.task-text(text=' ::task.text', remove-watch='true', use-timeout=':: $index > 10', timeout-time=':: ($index > 10) ? $index * 7 : 50')
```

Comment:
Notes: On the first ten tasks the markdown with be parsed immediately, after that all other "invisible" tasks get to be parsed in an increasing timeout , which speeds up the list generation